pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -230,6 +230,13 @@ type PostFilterPlugin interface {
 	PostFilter(pc *PluginContext, pod *v1.Pod, nodes []*v1.Node, filteredNodesStatuses NodeToStatusMap) *Status
 }
 
+type ScoreExtensions interface {
```

Comment:
Need a comment here to fix golint failure

Reply:
Fixed